returnn/tf/network.py

```diff
@@ -949,14 +949,18 @@ def _create_layer(self, name, layer_class, **layer_desc):
           print("layer %s/%r output: %r" % (self.name, name, output_template))
         output_template.sanity_check(ignore_placeholder=True)  # placeholder might be overwritten later
         output_template_special_axes = output_template.get_special_axes_dict()
+        placeholder_is_available = True
```

Comment:
Why do we need this variable? Can't we just check for `layer.output.placeholder is not None` below?

Reply:
Fixed now.